client/.eslintrc.yml

```diff
@@ -23,6 +23,7 @@ rules:
   ### Stylistic issues
   indent: [1, 2, { SwitchCase: 1, VariableDeclarator: 2 }]
   id-length: [1, { min: 2, exceptions: [_, e, i, k, v] }]
+  space-before-function-paren: [1, "never"]
```

Comment:
@alexfedoseev @robwise @mapreal19 @mamute @dylangrafmyre @samnang @josiasds I like that AirBnb rule:

* Makes i clear when you're creating an anonymous function.
* Makes it clear that function() is really not a function named "function".